pandas/tests/io/json/test_ujson.py

```diff
@@ -559,6 +560,15 @@ def test_encode_long_conversion(self):
         assert output == json.dumps(long_input)
         assert long_input == ujson.decode(output)
 
+    def test_dumps_ints_larger_than_maxsize(self):
```

Comment:
can you also add a full integration test in json/test_pandas.py

Reply:
Should also parametrize this to exceed the minimum supported native size

Reply:
Can you do the parametrization of a large negative value here too?